### DISCUSSION TITLE
Throttling Rate somehow available?

### DISCUSSION THREAD

**Discussion by Sven177** [syssi/esphome-modbus-solax-x1#67] [Q&A]
Hi,
the Gateway for Export Control still works like a charm 😃

I was just wondering if along all those data, available on a Solax Inverter, there would be the actual throttling rate available as well? 🤔
For this would be a perfect way to determine if, and how much, of surplus energy would be available when not throttling down.
So let´s say there is a consumer with a certain wattage, you only want to run without any power import from the grid. OK - switching it on will let you see immediately, if you still need additional power from the grid or not 😉 - but it would be nice if one could know about the current throttling rate, to calculate (or at least estimate) the maximum power available. Just to know upfront, if full power performance would be sufficient, or if still power from the grid would be needed.

Is such information even available from the Inverter?
And if not, then of course just forget about that idea... 😅

Regards!
Sven 🙋🏼‍♂️👋🏼

**Comment by syssi** [comment]
In "Export control" mode no measurements are available via RS485 unfortunately. You could use the OEM WiFi dongle to collection some details here. You can probably approximate the available solar energy by multiplying `pv_voltage * pv_current * efficiency`.

**Comment by Sven177** [comment]
OK - thanks for the quick feedback... ☺️
